addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -83,17 +83,17 @@
 		<item-type>Number</item-type>
 		<label>Total Usage</label>
 		<description>Used energy in Watt hours</description>
-		<state pattern="%d Wh" readOnly="true"></state>
+		<state pattern="%f Wh" readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="instantamp" advanced="true">
 		<item-type>Number</item-type>
 		<label>Instant Amp</label>
 		<description>Instant current in Amperes</description>
-		<state pattern="%d A" readOnly="true"></state>
+		<state pattern="%f A" readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="totalamphours" advanced="true">
```

Comment:
no particular reason other than i knew which bits i had had to change by looking at the code quickly (i.e. if singular i had changed it), ive updated the PR and have made it all singular, having only just spotted this. if you really want it converting back to multiples it can be done, but as you say no functionality is lost either way.